.circleci/config.yml

```diff
@@ -145,19 +124,49 @@ jobs:
           destination: result
 
   flake8:
-    <<: *test_defaults
-    environment:
-      TOX_TEST: flake8
+    docker:
+      - image: circleci/python:2.7
```

Comment:
Do we not want to standardise on the same image? (2.7.14 above)

Reply:
no, the above things are an exception, we don't downgrade unless there's a good reason

Reply:
OK, makes sense- is it worth adding a comment stating why we deliberately downgrade the above one? (I know we discussed why earlier)